lm75bd/lm75bd.c

```diff
@@ -25,15 +25,64 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
-error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
-  return ERR_CODE_SUCCESS;
+/*
+ * @brief Reads temperature from LM75BD
+ *
+ * @param devAddr - Address of the device to communicate to  
+ * @param temp - Float to store temperature in
+ *
+ */
+error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {  
+    float result = 0.0;
```

Comment:
nit: since this variable isnt used until the end of the function I would declare it down there instead of up here